Azure_test_file.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
Issue - Ensure that logging for Azure KeyVault is 'Enabled' 
 category - Logging and Monitoring 
 severity - HIGH